src/style.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  height: 100px;
+  width: 100px;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--0 {
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
```

Comment:
Not needed here. You've already applied defauld background params to `.stars__star` class,